README.md

```diff
@@ -1,3 +1,7 @@
+<a href="https://travis-ci.org/gthomas2/moodle-filter_imageopt">
+<img src="https://travis-ci.org/gthomas2/moodle-filter_imageopt.svg?branch=master">
```

Comment:
Please add four spaces before the img tag and then I'll merge in the PR. Thanks for the contribution!

Reply:
I've fixed this, but please don't merge this yet, there is still something weird with travis, both the patch itself isn't quite right but also I'm not actually seeing the travis builds linked to the pr's like I normally would

Reply:
Ok I've got travis working correctly now, so I'm happy for this to merge when ever you are.

Reply:
Merged, thank you for your contribution :-)